### ISSUE TITLE
ParallelCluster 3.8.0b1 does not consistently mount all shared file systems via EFS when configured

### ISSUE THREAD

**Issue by michaelmayer2** [aws/aws-parallelcluster#5812]
When using `InternalSharedStorageType: Efs` with AWS PC 3.8.0b1, 

- /opt/parallelcluster/shared
- /opt/parallelcluster/shared_login_nodes
- /opt/slurm 
- /opt/intel

are exported to the login Nodes via EFS but 

- /opt/parallelcluster/scripts/login_nodes

is still exported by the head node via NFS and mounted on the login nodes. 

On a login node I can see 

```
LABEL=cloudimg-rootfs	/	 ext4	defaults,discard	0 1
LABEL=UEFI	/boot/efi	vfat	umask=0077	0 1
fs-xxx.efs.eu-west-1.amazonaws.com:/opt/parallelcluster/shared /opt/parallelcluster/shared efs _netdev,noresvport 0 0
fs-xxx.efs.eu-west-1.amazonaws.com:/opt/parallelcluster/shared_login_nodes /opt/parallelcluster/shared_login_nodes efs _netdev,noresvport 0 0
fs-xxx.efs.eu-west-1.amazonaws.com:/opt/slurm /opt/slurm efs _netdev,noresvport 0 0
fs-xxx.efs.eu-west-1.amazonaws.com:/opt/intel /opt/intel efs _netdev,noresvport 0 0
a.b.c.d:/home /home nfs hard,_netdev,noatime 0 0
a.b.c.d:/opt/parallelcluster/scripts/login_nodes /opt/parallelcluster/scripts/utils nfs hard,_netdev,noatime 0 0
fs-xxx.efs.eu-west-1.amazonaws.com:/ /opt/parallelcluster/init_shared efs _netdev,noresvport 0 0
fs-xxx.fsx.eu-west-1.amazonaws.com@tcp:/htmsvbev /shared lustre defaults,_netdev,flock,user_xattr,noatime,noauto,x-systemd.automount 0 0
```

while on the head node I can see in /etc/exports 

```
/opt/parallelcluster/scripts/login_nodes x.y.0.0/16(rw,sync,no_root_squash)
/home x.y.0.0/16(rw,sync,no_root_squash)
```

**Comment by davprat** [comment]
Thank you for your report. I've added this to our internal tracking systems.

**Comment by dreambeyondorange** [comment]
Hi, this issue is addressed by the following PR that will be included in the actual 3.8 release https://github.com/aws/aws-parallelcluster-cookbook/pull/2522
